src/config/feature-flags.ts

```diff
@@ -14,15 +14,9 @@ const launchdarklyClient = LaunchDarkly.init(envVars.LAUNCHDARKLY_KEY || "", {
 
 export enum BooleanFlags {
 	MAINTENANCE_MODE = "maintenance-mode",
-	EXPOSE_QUEUE_METRICS = "expose-queue-metrics",
-	PROCESS_PUSHES_IMMEDIATELY = "process-pushes-immediately",
 	SIMPLER_PROCESSOR = "simpler-processor",
-	NEW_GITHUB_ERROR_PAGE = "new-git-hub-error-page",
-	NEW_BACKFILL_PROCESS_ENABLED = "new-backfill-process-enabled",
```

Comment:
These 3 constants were unused:
EXPOSE_QUEUE_METRICS
NEW_GITHUB_ERROR_PAGE
NEW_BACKFILL_PROCESS_ENABLED

Apparently these flags were removed from the code a while ago